add an nidaqmx example with an fft (#210)
ni/nipanel-python: examples/nidaqmx/nidaqmx_analog_input_fft/README.md

> ## Overview

Reviewer: ```suggestion
# NI-DAQmx Analog Input FFT Example

### Overview
```

> Prerequisites

Reviewer: See comments on #211

This is a nipanel example that displays an interactive Streamlit app and updates continuous analog input samples while measuring and displaying the FFT Spectrum Magnitude.

### Features

- NI-DAQmx Python configuration and acquisition
- Displays data in an interactive chart using ECharts
- Updates automatically as new data is acquired

### Prerequisites

- Python 3.10 or later
- InstrumentStudio 2026 Q1 or later

> - A Physical or Simulated Device : https://github.com/ni/nidaqmx-python/blob/master/README.rst (Getting Started Section)

Reviewer: ```suggestion
- A physical or simulated device: https://github.com/ni/nidaqmx-python/blob/master/README.rst (Getting Started section)
```

# NI-DAQmx Analog Input with FFT Example

### Overview

> This is a nipanel example that displays an interactive Streamlit panel and updates continuous analog input samples while measuring and displaying the FFT Spectrum Magnitude.

Reviewer: "updates"?

"Spectrum Magnitude" should not be capitalized. Also consider rewording to use "time domain" and "frequency domain", or at least mention the non-FFT graph in some way.

```suggestion
This is a nipanel example that acquires a continuous amount of data using the DAQ device's internal clock and graphs the acquired data in both the time domain and frequency domain in an interactive Streamlit panel.```
